core/src/main/java/nu/yona/server/messaging/service/MessageDto.java

```diff
@@ -186,9 +186,7 @@ protected final SenderInfo getSenderInfo(UserDto actingUser, Message messageEnti
 					return createSenderInfoForSelf(actingUser);
 				}
 
-				// Note: actingUser may be out of sync here, because the message action may have changed its state
```

Comment:
No, it's fine with me. If the warning isn't applicable anymore, we shouldn't have it in the code.